tests/fuzz/s2n_client_cert_recv_test.c

```diff
@@ -42,6 +45,16 @@ static void s2n_fuzz_atexit()
 
 int LLVMFuzzerInitialize(const uint8_t *buf, size_t len)
 {
+#ifdef S2N_TEST_IN_FIPS_MODE
```

Comment:
Can we move these repeated #ifdef's to s2n_test.h and into a `ENTER_FIPS_MODE()` macro?